server/notification-providers/wecom.js

```diff
@@ -30,27 +31,66 @@ class WeCom extends NotificationProvider {
     /**
      * Generate the message to send
      * @param {object} heartbeatJSON Heartbeat details (For Up/Down only)
+     * @param {object} monitorJSON Monitor details
      * @param {string} msg General message
      * @returns {object} Message
      */
-    composeMessage(heartbeatJSON, msg) {
-        let title;
-        if (msg != null && heartbeatJSON != null && heartbeatJSON["status"] === UP) {
-            title = "UptimeKuma Monitor Up";
-        }
-        if (msg != null && heartbeatJSON != null && heartbeatJSON["status"] === DOWN) {
-            title = "UptimeKuma Monitor Down";
-        }
-        if (msg != null) {
-            title = "UptimeKuma Message";
+    composeMessage(heartbeatJSON, monitorJSON, msg) {
+        if (heartbeatJSON != null) {
+            return {
+                msgtype: "template_card",
+                template_card: {
+                    card_type: "text_notice",
+                    main_title: {
+                        title: this.statusToString(
+                            heartbeatJSON["status"],
+                            monitorJSON["name"]
+                        ),
+                    },
+                    sub_title_text: heartbeatJSON["msg"],
+                    horizontal_content_list: [
+                        {
+                            keyname: "Timezone",
+                            value: heartbeatJSON["timezone"],
+                        },
+                        {
+                            keyname: "Time",
+                            value: heartbeatJSON["localDateTime"],
+                        },
+                    ],
+                    card_action: {
+                        type: 1,
+                        url: monitorJSON["url"]
+                            ? monitorJSON["url"]
+                            : "https://",
```

Comment:
<details><summary>That is super strange, but you are obviously right.</summary>
<p>

![image](https://github.com/louislam/uptime-kuma/assets/26258709/3e24cdf7-286e-4eed-803c-463deff6850c)

</p>
</details> 

I think having a more meaningful url here is still better than nothing

```suggestion
                            : "https://github.com/louislam/uptime-kuma", // both card_action and card_action.url are mandatory
```